### DISCUSSION TITLE
Генерация голосовых файлов Я.Алисой через демо страницу под капотом

### DISCUSSION THREAD

**Discussion by 112358aalexx** [mikopbx/Core#58] [Ideas (Идеи)]
Добрый день! Сейчас в продукте есть две возможности формирования голосовых файлов в интерфейсе, предлагаю добавить третью. Вынести её в отдельную вкладку и дополнить некоторыми описаниями(советами) разметки текста для озвучивания. 
Получаемый от Яндекса звуковой файл подтягивать давать на прослушивание во всплывающем на текущей странице html5 плеере и по нажатию кнопки Одобрить помещать во вкладку с основными звуковыми файлами системы.

**Comment by boffart** [comment]
Через демо страницу не выйдет. Там авторизация по токену обязательна. 
Тема тоже интересная, недавно запилили модуль модуль, разворачивающий RHVoice в локальном docker, отлично генерирует речь. 

Опять же напрашивается доработка "Источников генерации речи", это может быть не только Yandex, но и Google / ЦРТ / Амазон и другие. 

Планы есть, но не ближайшие. Задача второстепенная.
